webboot/webboot.go

```diff
@@ -117,12 +118,24 @@ func usage() {
 	os.Exit(1)
 }
 
+func setupWifi(wifi string) error {
+	o, err := exec.Command("iwconfig", "wlan0", "essid", wifi).CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("error setting up wifi:%v", err)
+	}
+	log.Printf("Connected to Wifi Succesfully %v", string(o))
```

Comment:
what if the command ran but iwconfig had an error anyway? 
Maybe you could do
log.Printf("setupWifi: iwconfig returned %v", string(o))

Reply:
Should there be a print in the function anyway? Wouldn't this be an issue for testing or other things? Instead, could there not be a print statement in main() after the function runs successfully? By then, wifi would be successfully connected and all of the errors would be handled before the print message in main()

Reply:
Yeah I think there should be a print in here because we need to make sure we received the correct message from running iwconfig.Any output from iwconfig should be printed out to enable us to debug in case of an error or connection failure.